### DISCUSSION TITLE
How do I get a FULL keycloak.init() response?

### DISCUSSION THREAD

**Discussion by SoozaV** [keycloak/keycloak#23266] [Q&A]
when I use the keycloak.init() I only get an 'authenticated' boolean on `then` function >

`keycloak.init(initOptions).then(authenticated => { /* more code */ })`

but in the network tab of the developer tools I can see a token request with a full response:

![image](https://github.com/keycloak/keycloak/assets/23610892/837fc174-742f-4a94-990d-e59bb08edf41)
I need to store all this values, especially the `refresh_expires_in` value:
![image](https://github.com/keycloak/keycloak/assets/23610892/b2667802-9af9-43f5-883f-d8ab410282c3)

the keycloak object doesn't have any prop or method for retrieve such data.

how can I do that?

Regards.

**Comment by jonkoops** [comment]
There are various properties and methods on the Keycloak instance that you can use to retrieve that information, for a complete overview you can consult [the TypeScript definition](https://github.com/keycloak/keycloak/blob/main/js/libs/keycloak-js/dist/keycloak.d.ts#L345).
